flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -169,29 +171,44 @@ public void testUpsertAndQuery() {
   public void testPrimaryKeyEqualToPartitionKey() {
```

Comment:
I feel we probably don't need to change this test method. there is no need to use the same schema as the previous method. there is an extra column for cognitive burden.